src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  overflow: hidden;
+  position: relative;
+  border: 1px solid $color-card-border;
+  transition-duration: 300ms;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    margin-inline: auto;
+    background-repeat: no-repeat;
+    width: 160px;
+    height: 134px;
+    background-position: center;
+    background-size: contain;
+  }
+
+  &__info {
+    @include roboto-medium-500;
+
+    margin-top: 40px;
+    font-size: $font-size-medium;
+    line-height: $line-height-medium;
+    color: $color-main;
+  }
+
+  &__code {
+    @include roboto-regular-400;
+
+    margin-top: 4px;
+    font-size: $font-size-small;
+    line-height: $line-height-small;
+    color: $color-secondary;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: center;
+
+    &-stars {
+      @include roboto-regular-400;
+
+      font-size: $font-size-small;
+      color: $color-main;
+    }
+
+    &-count {
+      @include roboto-regular-400;
+
+      font-size: $font-size-small;
+      color: $color-main;
+    }
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-title {
+      @include roboto-regular-400;
+
+      font-size: $font-size-medium;
+      color: $color-secondary;
+    }
+
+    &-value {
+      @include roboto-bold-700;
+
+      color: $color-main;
+    }
+  }
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &:hover .card__info {
+    color: #34568b;
```

Comment:
The hover effect changes the text color to #34568b. Ensure that this color provides sufficient contrast against the background for accessibility. Consider using a tool to check color contrast ratios.